opendevin/schema/config.py

```diff
@@ -7,6 +7,7 @@ class ConfigType(str, Enum):
     WORKSPACE_BASE = 'WORKSPACE_BASE'
     WORKSPACE_MOUNT_PATH = 'WORKSPACE_MOUNT_PATH'
     WORKSPACE_MOUNT_REWRITE = 'WORKSPACE_MOUNT_REWRITE'
+    WORKSPACE_MOUNT_PATH_IN_SANDBOX = 'WORKSPACE_MOUNT_PATH_IN_SANDBOX'
```

Comment:
How is this config different from `WORKSPACE_MOUNT_PATH`?

Reply:
`WORKSPACE_MOUNT_PATH` is the directory of workspace folder on the host machine, and `WORKSPACE_MOUNT_PATH_IN_SANDBOX` is the folder path inside the docker sandbox.

something like `docker run -v $WORKSPACE_MOUNT_PATH:$WORKSPACE_MOUNT_PATH_IN_SANDBOX ...`

Reply:
This one!

https://github.com/OpenDevin/OpenDevin/blob/d692a72bf3809df35d802041211fcd81d56b1dc6/opendevin/sandbox/docker/ssh_box.py#L307